packages/amplify-provider-awscloudformation/lib/delete-env.js

```diff
@@ -33,6 +34,7 @@ async function getStorageBucket(context, envName, s3) {
   const amplifyMeta = context.amplify.readJsonFile(`${unZippedDir}/amplify-meta.json`);
   const storage = amplifyMeta['storage'] || {};
   const s3Storage = Object.keys(storage).filter(r => storage[r].service === 'S3');
+  fs.removeSync(tempDir);
```

Comment:
A function called "getStorageBucket" shouldn't have a side effect of deleting a directory.

I would refactor this function to take in the tempDir and then the calling function should compute the tempDir, call this function, then delete the tempDir

Reply:
fixed